config/settings.go

```diff
@@ -42,6 +42,7 @@ type NettestConfig struct {
 // Websites test group
 type Websites struct {
 	EnabledCategories []string `json:"enabled_categories"`
+	Limit			  int      `json:"limit"`
```

Comment:
Did you run `go fmt`?

Reply:
No. I need to check why my editor is not doing this on save anymore.